pkgs/development/python-modules/python-toolbox/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, nose
+, docutils
+}:
+
+buildPythonPackage rec {
+  version = "0.9.4";
+  pname = "python_toolbox";
```

```suggestion
  pname = "python_toolbox";
  disabled = isPy27;
```

looks like they don't support it:
```
builder for '/nix/store/cqj58n5ihgxkkmmk0qxr9rmjnqwph0z4-python2.7-python_toolbox-0.9.4.drv' failed with exit code 1; last 10 log lines:
    File "/build/source/source_py2/test_python_toolbox/test_future_tools/test_future_tools.py", line 4, in <module>
      import concurrent.futures
  ImportError: No module named concurrent.futures

  ----------------------------------------------------------------------
  Ran 522 tests in 199.426s

  FAILED (errors=1, skipped=7)
  Test failed: <unittest.runner.TextTestResult run=522 errors=1 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=522 errors=1 failures=0>
cannot build derivation '/nix/store/pzk44l92xshsmzasq76vd6y1c4c72jqa-python2.7-pysnooper-0.3.0.drv': 1 dependencies couldn't be built
```

---

Ah, thanks. I've also disabled tests on pysnooper with same condition :+1: